CMakeLists.txt

```diff
@@ -486,6 +493,16 @@ if(VELOX_ENABLE_WAVE OR VELOX_ENABLE_CUDF)
     set(VELOX_ENABLE_ARROW ON)
     velox_set_source(cudf)
     velox_resolve_dependency(cudf)
+    if(TARGET aws-cpp-sdk-core)
```

Comment:
This should be based on whether we are bundling curl and not based on WAVE / CUDF.
I will clean up here https://github.com/facebookincubator/velox/pull/14160